src/main/java/org/jitsi/jicofo/FocusManager.java

```diff
@@ -258,6 +264,20 @@ public void start()
         DomainBareJid xmppDomain = JidCreate.domainBareFrom(
                 config.getString(XMPP_DOMAIN_PNAME));
 
+        int jicofoId = config.getInt(JICOFO_SHORT_ID_PNAME, -1);
+        if (jicofoId < 1 || jicofoId > 0xffff)
+        {
+            logger.warn(
+                "Jicofo ID is not set. Configure a valid value [1-65535] by "
+                + "setting " + JICOFO_SHORT_ID_PNAME + ". Future versions "
+                + "will require this for Octo.");
+            this.jicofoId = 0;
```

Comment:
Is this gonna be spitting out warnings on default-install jitsi-meet Debian instances?  Is there any way to detect the case where the system only has a single Jicofo, so you don't have to worry about this?

Reply:
It will log a single warning at startup, and any time a conference uses Octo.

I can't think of a way to detect when a system has more than one jicofo. We could provision debian/docker with a valid default valid to  suppress the log, but that defeats the purpose of the log -- if you're building a system with Octo and multiple jicofos you will not see the warning.